docs/targets/index.md

```diff
@@ -1,9 +1,11 @@
-# Event targets
+# Event Targets
 
-The following is a list of Triggermesh event Destinations known as Targets. Some are available as open source projects some are available as hosted solutions on our Cloud.
+The following is a list of Triggermesh event Destinations known as `Targets`. Some are available as open source projects some are available as hosted solutions on our Cloud.
```

Comment:
This is a run-on sentence, and should be corrected.

Reply:
Lets fix this in another PR or at the end because thats going to cause merge conflicts for all the other ones :)